tests/test_shopcarts.py

```diff
@@ -1,32 +1,46 @@
 """
-Test cases for YourResourceModel Model
+Test cases for Shopcart Model
 
 """
 import os
 import logging
 import unittest
-from service.models import YourResourceModel, DataValidationError, db
+from service.models import Shopcart, DataValidationError, db
+from service import app
 
+DATABASE_URI = os.getenv(
+    "DATABASE_URI", "postgresql://postgres:postgres@postgres:5432/postgres"
+)
 
 ######################################################################
-#  YourResourceModel   M O D E L   T E S T   C A S E S
+#  S H O P C A R T   M O D E L   T E S T   C A S E S
 ######################################################################
-class TestYourResourceModel(unittest.TestCase):
-    """ Test Cases for YourResourceModel Model """
+class TestShopcart(unittest.TestCase):
+    """ Test Cases for Shopcart Model """
 
     @classmethod
     def setUpClass(cls):
         """ This runs once before the entire test suite """
+        app.config["TESTING"] = True
+        app.config["DEBUG"] = False
+        app.config["DATABASE_URI"] = DATABASE_URI
+        app.logger.setLevel(logging.CRITICAL)
+        Shopcart.init_db(app)
 
     @classmethod
     def tearDownClass(cls):
         """ This runs once after the entire test suite """
+        db.session.close()
```

Comment:
as mentioned in Professor's video, I think we can remove the method `tearDownClass`: https://youtu.be/iZ2ogEx5hNQ?t=1411

Reply:
Agree, do you think it is better to annotate this function instead of deleting?

Reply:
agree! then let's leave it in code as a method that does nothing~